extensions/amp-sticky-ad/1.0/amp-sticky-ad.js

```diff
@@ -71,23 +67,11 @@ class AmpStickyAd extends AMP.BaseElement {
     this.ad_ = children[0];
     this.setAsOwner(this.ad_);
     whenUpgradedToCustomElement(dev().assertElement(this.ad_)).then(ad => {
-      if (!ad.whenBuilt) {
-        // TODO(@zhouyx, #9126): Cleanup once make sure the fix works
-        dev().error(TAG, 'element whenBuilt still do not exist!');
-        timerFor(this.win).delay(() => {
-          ad.whenBuilt().then(() => {
-            this.mutateElement(() => {
-              toggle(this.element, true);
-            });
-          });
-        }, 1000);
-      } else {
-        ad.whenBuilt().then(() => {
-          this.mutateElement(() => {
-            toggle(this.element, true);
-          });
+      ad.whenBuilt().then(() => {
```

Comment:
I see, changed! Thanks for the explanation!  However I don't think we have a style requirement for this, and I can find the second usage in a lot of places😢